### PR TITLE
Update using-admin-roles.md

### PR DESCRIPTION
**Pull request by pupara**
Updated for Teams PowerShell Module 2.0

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/using-admin-roles.md [hunk 1]
```diff
@@ -61,16 +61,13 @@ You can assign users to these roles in Azure AD. To learn how to assign administ
 
 ## Cmdlets available for each role
 
-Most of the PowerShell tools for these admin roles live in the Skype for Business PowerShell module, and it's important to note that some of the cmdlets that these admin roles have access to control shared settings that are also used for Skype for Business Online. The Skype for Business admin role also has access to all the cmdlets in the Skype for Business PowerShell module.
+Most of the PowerShell tools for these admin roles live in the Teams PowerShell module, and it's important to note that some of the cmdlets that these admin roles have access to control shared settings that are also used for Skype for Business Online. 
 
-To view the full list of cmdlets currently available to a given role in the Skype for Business PowerShell module, follow these steps:
+To view the full list of cmdlets, follow these steps:
```
Comment (JohanFreelancer9) [inline Teams/using-admin-roles.md:66]:
```suggestion
To view the full list of cmdlets:
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - suggestion.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**